data/human/hails.txt

```diff
@@ -2082,6 +2082,98 @@ phrase "friendly civilian"
 		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
 		"New Holland. It gets too boring there, though, so I fly the stars instead."
 		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
+		
+phrase "pyramid_scheme_company"
+	word
+		"Magellan "
+		"Dehydrated "
+		"Pontificated "
+		"General "
+		"Sacrosanct "
+		"Liberated "
+		"Literal "
+		"Family "
+		"Hyper-real "
+	word
+		"Dynamix"
+		"Unlimited"
+		"Orbital"
+		"Gigalight"
+		"Traders"
+		"Voidcraft"
+		"Whispers"
+
+phrase "friendly civilian"
+	word
+		"Hi! Good to see you, uh, "
+		"Well, if it isn't "
+		"Hi there, uhm, "
+		"I've been trying to get a hold of you, "
+	word
+		"Captain <Last>. "
```

Comment:
How do they know our name?

Reply:
I had assumed that your IFF transponder would have that kind of basic information.

Reply:
```suggestion
		"Captain <last>. "
```
Needs to be lowercase.